tests/manifold/projection_manifold_01.cc

```diff
@@ -0,0 +1,74 @@
+//-------------------------------------------------------------------
+//    Copyright (C) 2016 by the deal.II authors.
```

Comment:
i was just curios why there are different headers at all. One states that the file is part of `deal.II`, another does not. Both refer to `LGPL` but the text is different.